VaticanBAV/et/BAVet123.xml

```diff
@@ -0,0 +1,233 @@
+                                            <dim unit="leaf">7</dim>
+                                            <locus from="59r" to="65v"/>
+                                        </item>
+                                        <item xml:id="q10" n="9">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="66r" to="74v"/>
+                                        </item>
+                                        <item xml:id="q11" n="B">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="75r" to="76v"/>
+                                        </item>
+                                    </list>
+                                </collation>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="27"/>
+                            </layoutDesc>
+                        </objectDesc>
+                        
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <desc>More than one hand is involved in the writing.</desc>
+                                <date notBefore="1800" notAfter="1899"/>
+                            </handNote>
+                        </handDesc>
+                        
+                        <decoDesc>
+                            <decoNote type="diagram" xml:id="d1">
+                                <locus target="#66v"/>                                    
+                                <desc>
+                                    Diagram on the length of the shadow during the hours of the day for each month, entitled:
+                                    <q xml:lang="gez">በየወርኁ፡ ስፍረ<supplied reason="omitted">፡</supplied> ሰአት<supplied reason="omitted">፡</supplied></q>.
+                                </desc>
+                            </decoNote>
+                        </decoDesc>
+                        
+                        <bindingDesc>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1">Two wooden boards, partially covered with traces of parchment.</decoNote>
+                                <decoNote xml:id="b2" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="parchment"/>                                
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    
+                    <history>
+                        <origin>
+                            <origDate notBefore="1800" notAfter="1899"/>                       
+                        </origin>
```

Comment:
In the (very few) BAV manuscripts I encoded, I tried to extract from Tisserant's account whatever was relevant to the history of the manuscript in question, so as not to lose this information for users who do not read the printed catalogue. For example, for BAVet127, this was:
```
       <provenance>According to the account on the manuscripts’ provenance given by Tisserant, this manuscript is among those
                        acquired by <persName ref="PRS4805Grebaut" role="owner"/> in <placeName ref="LOC1216AddisA">ʾAddis ʾAbabā</placeName> 
                        in <date when="1925"/> with funds of
                        <persName ref="PRS14055SaintJ">Jean Saint</persName>. 
                        The manuscripts originated from Šawā (ʾAddis ʾAbabā, Dabra Libānos, Dabra Bǝrhān),
                        Goǧǧām (Dimā Giyorgis, Marṭula Māryām, Dabra Warq) and Bagemdǝr (Gondar, ʾAzazo Takla Hāymānot) and very few from Tǝgrāy.
                      </provenance>
                      <acquisition when="1928">The manuscript entered the library on <date>18 April 1928</date>.</acquisition>
```
In that case, I also added to the bibliographic references  
```
<listBibl type="secondary">
                                 <bibl>
                                   <ptr target="bm:GrebTiss1936Codices"/>
                                   <citedRange unit="page">20</citedRange>
                                 </bibl>
                               </listBibl>
```
I think that for many manuscripts this could be copy-pasted with only minor adaptations necessary. If it is not too much work, could you include such a paragraph here?

Reply:
Thank you for this information. I have added this to other manuscripts as well.